### PR TITLE
 Add tags & add 3 more resources to /design

### PR DESCRIPTION
**Pull request by S3B4S**
Wondering whether we should enforce plural/singular use of words for choice of tags. 🤔 What do you think? @lostdesign

### CODE REVIEW

**Comment by lostdesign** [comment]
I'd say singular use of words and max of three tags per resource?

File: resources/design.json [hunk 1]
```diff
@@ -6,139 +6,157 @@
...
       "url": "https://undraw.co",
-      "tags": []
+      "tags": ["illustration", "svg", "ui"]
     },
     {
       "title": "Practical UI tips",
       "desc": "7 Tips to boost your UI design.",
       "url": "https://medium.com/refactoring-ui/7-practical-tips-for-cheating-at-design-40c736799886",
-      "tags": []
+      "tags": ["ui", "tips", "tricks"]
```
Comment (lostdesign) [inline resources/design.json]: `tricks` is plural no? Singular in this case seems weird though.

Comment (S3B4S) [reply resources/design.json]: Yeah it is, and I agree that it sounds awkward.

**Comment by S3B4S** [comment]
Maybe it's unnecessary to enforce plural/singular usage of words.
> max of three tags per resource?
I'd say that's too little, thinking from the perspective of searching through tags something like `ui` might already also need the tags `user interface`. So the user will find the resource by both typing `ui` and `user i` (assuming we'll have a autocomplete dropdown).
I'd say a max of 5 tags should be sufficient, and at least 2 tags.

**Pull request merged**